### PR TITLE
Feature/implement trials screen functionality

### PR DESCRIPTION
**Pull request by kvinceto**

### CODE REVIEW

File: lib/enums/question_enum.dart [hunk 1]
```diff
@@ -0,0 +1,5 @@
+enum QuestionEnum {
+  radioButton,
+  checkbox,
+  freeText,
```
Comment (HristoPetkov94) [inline lib/enums/question_enum.dart:4]: inputText is better name

File: lib/models/question_model.dart [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import 'package:flutter_gp5/enums/question_enum.dart';
+
+class QuestionModel {
+  String questionText;
```
Comment (HristoPetkov94) [inline lib/models/question_model.dart:4]: it could be text only because the object name is making it clear

File: lib/models/question_model.dart [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  List<String> answers = [];
```
Comment (HristoPetkov94) [inline lib/models/question_model.dart:5]: dont we need the answer to be a single value not a list?

File: lib/screens/patient_trials/patient_trials_screen.dart [hunk 4]
```diff
@@ -1,10 +1,181 @@
...
+      appBar: AppBar(
+        title: const Text(
+          'My trials',
+          style: TextStyle(color: Colors.white),
+        ),
+        backgroundColor: Colors.teal.shade600,
+        centerTitle: true,
+        elevation: 4.00,
+      ),
+      body: _isLoaded
```
Comment (HristoPetkov94) [inline lib/screens/patient_trials/patient_trials_screen.dart:38]: for the body use FutureBuilder. Read what is it and how to use it. And wrap the __getTrials into a Future.value(_getTrials) object

**Review by HristoPetkov94 — CHANGES_REQUESTED** [review]

File: lib/enums/question_enum.dart (hunk 1 above)
Comment (kvinceto) [reply lib/enums/question_enum.dart]: done

File: lib/models/question_model.dart (hunk 2 above)
Comment (kvinceto) [reply lib/models/question_model.dart]: done

File: lib/screens/patient_trials/patient_trials_screen.dart (hunk 4 above)
Comment (kvinceto) [reply lib/screens/patient_trials/patient_trials_screen.dart]: done

File: lib/models/question_model.dart (hunk 3 above)
Comment (kvinceto) [reply lib/models/question_model.dart:5]: no, we have to get all possible answers for this question so we can visualize them

**Review by HristoPetkov94 — APPROVED** [review]

**Pull request merged**